panther/panther_hardware.repos

```diff
@@ -11,3 +11,7 @@ repositories:
     type: git
     url: https://github.com/husarion/husarion_controllers
     version: main
+  behaviortree_ros2:
+    type: git
+    url: https://github.com/BehaviorTree/BehaviorTree.ROS2
+    version: humble
```

Comment:
```suggestion
    version: ce923e19c12d28f4734a41b1442c123f2d4a41d2
```
Use fixed version of the repo. Since the last tag - 0.2.0 - there were service timeout fixes not sure if this affects our use case but think we may want stick to last known sha